lib/bundle/brew_dumper.rb

```diff
@@ -61,6 +56,18 @@ def expand_cask_requirements
 
     private
 
+    def self.formulae_info
+      @@formulae_info ||= begin
```

Comment:
I wonder do we need to cache this? Considering, it will only be run once for all of `brew bundle` subcommands. Beside by removing the cache, we can remove `formulae_info_reset!` at the same time.

If you want to keep it, you want to change `@@formulae_info` to `@formulae_info`